tests/test_resolutions.py

```diff
@@ -79,7 +83,7 @@ class MockResolution(BaseResolution):
 
 
 class TestContainsTheEntry:
-    def test_can_be_instantiated(self):
+    def test_can_be_instantiated(self) -> None:
```

Comment:
Haha i don't think we need type hints on the unit tests. 

I know it very quickly becomes habit. :P